config/settings.py

```diff
@@ -34,36 +34,36 @@
 
 # Application definition
 INSTALLED_APPS = [
-    "django.contrib.auth",
-    "django.contrib.contenttypes",
-    "django.contrib.sessions",
-    "django.contrib.messages",
-    "django.contrib.staticfiles",
-    "django_extensions",
-    "django.contrib.postgres",
-    "psqlextra",
-    "rest_framework",
-    "django_filters",
-    "osidb",
     "apps.bbsync",
     "apps.exploits",
     "apps.sla",
     "apps.trackers",
     "apps.workflows",
     "collectors.bzimport",
+    "collectors.epss",
     "collectors.errata",
+    "collectors.exploits_cisa",
+    "collectors.exploits_exploitdb",
+    "collectors.exploits_metasploit",
     "collectors.framework",
     "collectors.jiraffe",
+    "collectors.nvd",
     "collectors.product_definitions",
+    "corsheaders",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.messages",
+    "django.contrib.postgres",
+    "django.contrib.sessions",
+    "django.contrib.staticfiles",
+    "django_extensions",
+    "django_filters",
     "drf_spectacular",
+    "osidb",
     "polymorphic",
+    "psqlextra",
+    "rest_framework",
     "rest_framework_simplejwt",
```

Comment:
note that while it most likely won't be an issue at this point in time, the order of installed apps *does* matter: https://docs.djangoproject.com/en/3.2/ref/settings/#installed-apps

Reply:
Thanks, I was not aware of this. So I should I revert it or leave it in the original order just to be sure?